migration/v0.15.x-v0.16.x/migrate-crd.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+here="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+
+get_patch() {
+  local release_name=$1
+  local release_namespace=$2
+  local chart=$3
+
+  echo \''{"metadata":'\
+          '{ "annotations":'\
+            '{ "meta.helm.sh/release-name": "'"${release_name}"'",' \
+              '"meta.helm.sh/release-namespace": "'"${release_namespace}"'"},' \
+            '"labels":' \
+              '{ "app.kubernetes.io/managed-by": "Helm",'\
+                '"helm.sh/chart": "'"${chart}"'"}}}'\'
+}
+
+# WC
+
```

Comment:
Shouldn't you have velero and prometheus in WC as well?

Reply:
Also, for the duplicates you could have 
```
# Variables
RELEASE_NAME="cert-manager"
RELEASE_NAMESPACE="cert-manager"
CHART="cert-manager-v1.1.0
CRDS=(  certificaterequests.cert-manager.io      
    ...
)

# wc
for CRD in "${CRDS[@]}"; do
      "${here}/../../bin/ck8s" ops kubectl wc patch crd "${CRD}" --patch "$(get_patch "${RELEASE_NAME}" ${RELEASE_NAMESPACE}" "${CHART}")"
done
# sc
for CRD in "${CRDS[@]}"; do
      "${here}/../../bin/ck8s" ops kubectl sc patch crd "${CRD}" --patch "$(get_patch "${RELEASE_NAME}" ${RELEASE_NAMESPACE}" "${CHART}")"
done

```
Instead of specifying the variables twice.

Reply:
Hmm, I seemed to not have enabled velero in WC (strange, I can't recall disabling that :thinking: )
But I'll update it!

Reply:
Still missing prometheus in WC